test/test_docannotate.py

```diff
@@ -425,3 +425,33 @@ def __init__(self, arg: str):
 
     assert Demo.__init__.metadata.annotated_params['arg'].type_name == 'str'
     assert DemoAnn.__init__.metadata.annotated_params['arg'].type_class == str
+
+
+def test_custom_type_class():
+
```

Comment:
All tests should have a one line docstring at least indicating what the purpose of the test is.

Reply:
yeah missed..  Adedd - 3f11e55c64c